guides/issuing-assets.md

```diff
@@ -39,7 +39,7 @@ In the simplest situations, you can issue assets from your everyday Stellar acco
 
 ### Requiring or Revoking Authorization
 
-Accounts have [several flags](concepts/accounts.md#flags) related to issuing assets. You’ll almost always want to make sure your issuing account has the [`AUTHORIZATION REVOCABLE` flag set](concepts/assets.md#revoking-access). This allows you to freeze your assets in another account.
+Accounts have [several flags](concepts/accounts.md#flags) related to issuing assets. You might want to make sure your issuing account has the [`AUTHORIZATION REVOCABLE` flag set](concepts/assets.md#revoking-access). This allows you to freeze your assets in another account.
```

Comment:
Ok. How about:

> Accounts have [several flags](concepts/accounts.md#flags) related to issuing assets. Setting the [`AUTHORIZATION REVOCABLE` flag](concepts/assets.md#revoking-access) allows you to freeze assets you issued in case of theft or other special circumstances. This can be useful for national currencies, but is not always applicable to other kinds of assets.

Does that work? Should we enumerate some of the specific situations noted above?